config/config.go

```diff
@@ -1,11 +1,19 @@
 package config
 
 import (
+	"github.com/ONSdigital/dp-feedback-api/sdk"
 	"time"
 
 	"github.com/kelseyhightower/envconfig"
 )
 
+// Config for feedback sdk
+type FeedbackConfig struct {
+	BindAddr         string
+	ServiceAuthToken string
+	Client           *sdk.Client
+}
+
 // Config represents service configuration for dp-frontend-feedback-controller
 type Config struct {
```

Comment:
if using a separate struct for the feedback api, then you can add it here as
`FeedbackAPI *FeedbackConfig`

Reply:
Done.